sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Configuration
+subtitle: How configure nf-core pipelines
+shortTitle: Configuration options
+weight: 1
+parentWeight: 20
+---
+
+## Configure nf-core pipelines
+
+Each nf-core pipeline comes with a set of “sensible defaults”.
```

Comment:
In regards to waht exactly? Memory resources? Parameters?

Reply:
I think it's important what to use them for

Reply:
Now framed “sensible defaults” for a typical analysis of full size data. Will expand below